examples/AccountManagement/GetAccountHierarchy.php

```diff
@@ -28,23 +28,29 @@
 use Google\Ads\GoogleAds\Lib\V2\GoogleAdsException;
 use Google\Ads\GoogleAds\V2\Errors\GoogleAdsError;
 use Google\Ads\GoogleAds\V2\Resources\CustomerClient;
+use Google\Ads\GoogleAds\V2\Services\CustomerServiceClient;
 use Google\Ads\GoogleAds\V2\Services\GoogleAdsRow;
 use Google\ApiCore\ApiException;
 
 /**
- * This example gets the account hierarchy of the specified customer account.
+ * This example gets the account hierarchy of the specified manager account. If you don't specify
+ * manager customer ID, the example will instead print the hierarchies of all accessible customer
+ * accounts for your authenticated Google account.
+ * Note that if your accessible customers have manager-client relationships, you may get hierarchies
```

Comment:
Instead of "have manager-client relationships...", please make it more general to cover the case where the user has direct access to a sub-manager as well, e.g.,

Note that if the list of accessible customers for your authenticated Google account includes accounts within the same hierarchy, this example will retrieve and print the overlapping portions of the hierarchy for each accessible customer.